### ISSUE TITLE
Top 123 regions

### ISSUE THREAD

**Issue by alinapark** [1712n/yachay-public#3]
The first suggested methodology on training the model is to look into the dataset of 123 regions around the world, where each region is represented by 5,000 tweets. The dataset attached provides an annotated corpus of 500k texts, as well as the respective geocoordinates.

The data set is [here](https://drive.google.com/file/d/1J5ducw8O628wyXD7qdcvop2pyNbq26tO/view?usp=sharing)

![This is an image](https://media.tenor.com/lOPTx_JZJ3gAAAAC/the-office-steve-carell.gif)

**Comment by AnuravModak** [comment]
is this challenge still open and currently looking for contributions???? @alinapark @ingakaspar

**Comment by ingakaspar** [comment]
@AnuravModak yes! All challenges are open :) If you have questions/ideas/suggestions, do not hesitate to start a discussion [in our discord](https://discord.com/invite/msWFtcfmwe)

**Comment by AnuravModak** [comment]
Is there any user guide available explaining how to implement and use the model.. something like this [click here](https://scikit-learn.org/stable/modules/generated/sklearn.neighbors.KNeighborsClassifier.html),( i have attached the example of user-guide for knn from scikit-learn.) @ingakaspar
